docs/guides/admin/docs/configuration/database.md

```diff
@@ -103,10 +103,10 @@ Finally, leave the client and restart the database server to enable the new user
 % systemctl restart mariadb.service
 ```
 
-PostgreSQL
-----------
+Step 3: PostgreSQL
+------------------
 
-Opencast's official PostgreSQL support is still marked as experimental.
+Opencast's official PostgreSQL support is still marked as experimental. <!-- _How up-to-date is this? -->
```

Comment:
if so, there's at least one other spot (also in this doc) that needs to be updated as well. once the info is confirmed, i'll update it

Reply:
No, this is current. It's been a while since 9.x is half a year late but we wanted it to be experimental for 9.x to mark it stable one release later. You should be able to use it already in production. But people should be aware that they are kind of early adopters.